project/build.properties

```diff
@@ -1 +1,2 @@
+# also update .github/workflows/ci.yml to ensure compatibility of the plugin via scripted
```

Comment:
I see mergify is set up here, so I am not sure this comment really helps. Can mergify ignore that specific bump maybe?

Reply:
Nothing needs to be updated with mergify. The trick we're using is to create a single "ci-passed" job that runs at the very end only if all tests pass: https://github.com/thesamet/sbt-protoc/blob/adbb4484cd3e593425a2a0e11449a959d4d496f5/.github/workflows/ci.yml#L45-L49 and mergify awaits only on this.

Reply:
Sorry if I wasn't clear: what I meant is that the scala-steward sbt bumps will be merged automatically, so we won't get the  opportunity to keep the `scripted-sbt` array values up to date as sbt 1.4.x patch releases roll out, like this comment suggests.